modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java

```diff
@@ -892,27 +909,47 @@ private void _populateWithTasks(Document document, Instance instance) {
 					task.get("assigneeType"), User.class.getName())) {
 
 				for (Object assigneeId : (List<?>)task.get("assigneeIds")) {
-					Assignee assignee = AssigneeUtil.toAssignee(
-						_language, _portal,
-						ResourceBundleUtil.getModuleAndPortalResourceBundle(
-							contextAcceptLanguage.getPreferredLocale(),
-							InstanceResourceImpl.class),
-						GetterUtil.getLong(assigneeId),
-						_userLocalService::fetchUser);
-
-					if (assignee != null) {
-						assignees.add(assignee);
-					}
+					assignees.add(
+						AssigneeUtil.toAssignee(
+							_language, _portal,
+							ResourceBundleUtil.getModuleAndPortalResourceBundle(
+								contextAcceptLanguage.getPreferredLocale(),
+								InstanceResourceImpl.class),
+							GetterUtil.getLong(assigneeId),
+							_userLocalService::fetchUser));
 				}
 			}
 			else if (Objects.equals(
 						task.get("assigneeType"), Role.class.getName())) {
 
+				List<Long> assigneeGroupIds = null;
+
+				List<Long> assigneeIds = null;
+
+				Map<Long, List<Long>> mapGroupRoleIds = new HashMap<>();
+
 				boolean reviewer = false;
 
-				for (Object assigneeId : (List<?>)task.get("assigneeIds")) {
+				List<Long> roleIds;
+
+				if (task.get("assigneeGroupIds") != null) {
+					assigneeGroupIds = ListUtil.toList(
+						(List<?>)task.get("assigneeGroupIds"),
+						GetterUtil::getLong);
+				}
+
```

Comment:
@seltong, please, define the `assigneeIds` variable below this condition, thanks!